cmake/OmimHelpers.cmake

```diff
@@ -79,14 +79,26 @@ function(omim_add_library library)
 
   # Enable warnings for all our libraries.
   target_compile_options(${library} PRIVATE ${OMIM_WARNING_FLAGS})
-  target_include_directories(${library} PRIVATE ${OMIM_INCLUDE_DIRS})
+  target_include_directories(${library} SYSTEM PRIVATE ${OMIM_INCLUDE_DIRS})
   if (USE_PPROF AND PLATFORM_MAC)
     find_path(PPROF_INCLUDE_DIR NAMES gperftools/profiler.h)
     target_include_directories(${library} SYSTEM PUBLIC ${PPROF_INCLUDE_DIR})
   endif()
   if (USE_PCH)
     add_precompiled_headers_to_target(${library} ${OMIM_PCH_TARGET_NAME})
   endif()
+
```

Comment:
I assumed that all targets configured with `omim_add_library` are considered our "own" targets, but not thirdparty deps, but there are a couple of libraries from `3party` folder configured with this `omim_add_library` too, i.e. clang-tidy checks are applied to them too. As I understand we changed those libraries somehow and that's why consider them as "own"? Or I wrongly get purpose of `omim_add_library`?

Reply:
Many 3p libs are needed for OM. We add most of them with omim_add_library for convenience. Some libs in 3p, like opening hours, are also our own ones (it should be completely refactored btw and moved into the main source tree).